### PR TITLE
Feature/mb upgrade annofuse

### PR DESCRIPTION
**Pull request by migbro**
## Description

This is hotfix of sorts in which the references for annoFuse have been updated with latest gene symbols to properly annotate GENCODE39+ gene fusions. I took the liberty of modernizing slightly by putting the R scripts here and also dropping a step that actually did nothing to an input.

Fixes # https://github.com/d3b-center/bixu-tracker/issues/1808

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [Old output](https://cavatica.sbgenomics.com/u/kids-first-drc/kids-first-drc-rnaseq-workflow/files/63f51780eda9b024c7893ed3/)
- [ ] [New output](https://cavatica.sbgenomics.com/u/kids-first-drc/kids-first-drc-rnaseq-workflow/files/63f51759eda9b024c7893e5d/)
Note that gene `ZFTA` is missing an annotation for `Gene1A_anno` and the new version correctly annotates

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: workflow/kfdrc_RNAseq_workflow.cwl [hunk 1]
```diff
@@ -872,5 +872,5 @@ hints:
 - SE
 - STAR
 "sbg:links":
-- id: 'https://github.com/kids-first/kf-rnaseq-workflow/releases/tag/v4.3.0'
+- id: 'https://github.com/kids-first/kf-rnaseq-workflow/releases/tag/v4.3.1'
```
Comment (dmiller15) [inline workflow/kfdrc_RNAseq_workflow.cwl:875]: Is this really a bug fix? Seems more substantial.

**Review by dmiller15 — APPROVED** [review]

**Review by sakshamphul — APPROVED** [review]

File: workflow/kfdrc_RNAseq_workflow.cwl (hunk 1 above)
Comment (migbro) [reply workflow/kfdrc_RNAseq_workflow.cwl:875]: yeah, I was debating that a bit...probably enough to justify a 4.4.0

Comment (migbro) [reply workflow/kfdrc_RNAseq_workflow.cwl:875]: Like, under the hood a lot happened, from a user perspective, aside from the tools, their interaction with the wf is the same

Comment (dmiller15) [reply workflow/kfdrc_RNAseq_workflow.cwl]: Well that sounds like a minor update: `MINOR version when you add functionality in a backwards compatible manner`. Going to `4.4.0` is appropriate.

**Pull request merged**